plugins/extractors/mariadb/mariadb.go

```diff
@@ -0,0 +1,230 @@
+package mariadb
+
+import (
+	"context"
+	"database/sql"
+	_ "embed" // used to print the embedded assets
+	"fmt"
+	_ "github.com/go-sql-driver/mysql" // used to register the mariadb driver
```

Comment:
fix the driver here

Reply:
Mariadb supports the same driver as of mysql